app/templates/suppliers/edit_duns_number.html

```diff
@@ -0,0 +1,39 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+  DUNS number – {{ supplier.name }} – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": url_for('.index'),
+        "label": "Admin home"
+      },
+      {
+        "label": supplier.name,
+        "link": url_for('.supplier_details', supplier_id=supplier.id)
+      },
+      {
+        "label": "DUNS number"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+<div class="grid-row">
+<div class="column-two-thirds">
+  {% with heading = "Update {}’s DUNS number".format(supplier.name) %}
```

Comment:
Might be nicer to have these as

```suggestion
  {% with heading = "Update DUNS number for ‘{}’".format(supplier.name) %}
```

because then the name of the thing your editing is always in the same place (doesn't get pushed over by long supplier names) and we can put quotation marks around supplier name without worrying about apostrophes.

Reply:
Approved - have added a commit.